src/olympia/devhub/templates/devhub/index.html

```diff
@@ -65,7 +66,7 @@
     </div>
     {% block site_js %}
       <script src="{{ static('js/i18n/%s.js'|format(LANG)) }}"></script>
-      {{ js('common') }}
+      {{ js('devhub/new-landing/js') }}
```

Comment:
I took the opportunity to drop the common js bundle from the dev-landing pages - only thing that appeared to be needed from the mystery soup of js was code for the lang switcher, so I broke that out and included it separately.